nixos/modules/services/web-apps/mealie.nix

```diff
@@ -0,0 +1,108 @@
+{ config, lib, pkgs, ...}:
+let
+  cfg = config.services.mealie;
+  pkg = pkgs.mealie;
+in
+{
+  options.services.mealie = {
+    enable = lib.mkEnableOption "Mealie, a recipe manager and meal planner";
+
+    # TODO allow_signup
+
+    host = lib.mkOption {
+      type = lib.types.str;
+      default = "localhost";
+      description = "Host on which creating the URL on the service";
```

Comment:
Is it a listen address? Or for links created on the UI?

Reply:
Only links for the UI